happytransformer/happy_bert.py

```diff
@@ -39,15 +39,25 @@ class HappyBERT(HappyTransformer):
             """
 
     def __init__(self, model='bert-base-uncased'):
+        # todo remove model parameter. Each model will have its own
         super().__init__(model, "BERT")
         self.mlm = None  # Masked Language Model
         self.nsp = None  # Next Sentence Prediction
-        self.qa = None   # Question Answering
-        self.tokenizer = BertTokenizer.from_pretrained(model)
+
+        #todo separate tokenizer for each model
+        self.tokenizer = BertTokenizerFast.from_pretrained("bert-base-uncased")
         self.masked_token = self.tokenizer.mask_token
         self.sep_token = self.tokenizer.sep_token
         self.cls_token = self.tokenizer.cls_token
 
+        # ------------------------ QA
```

Comment:
The slew of `__qa_` variables seems weird. Holding a single class instance might be cleaner. Also, it would be more maintainable if this top level logic was stored in `happy_transformer.py`, with implementation specifics in `happy_bert.py`.

Reply:
I agree, that does seem cleaner. I did it this way because not every Happy class will have QA functionality, but on second thought you're right. Keeping them private and as None does no harm even if the class does not use them.